pytorch_lightning/trainer/trainer.py

```diff
@@ -2206,6 +2211,9 @@ def weights_save_path(self) -> str:
         """
```

Comment:
if someone acess this property, should a warning be raised here too?

Reply:
Unfortunately we can't do that, since we still need to use `trainer.weights_save_path` in the codebase until it is removed, like here: https://github.com/PyTorchLightning/pytorch-lightning/blob/a0655611de460f1659b46150cda256d2e3fa974e/pytorch_lightning/callbacks/model_checkpoint.py#L579-L592

I think it is sufficient to warn if someone passes `weights_save_path` to the Trainer as an argument.